src/app/static/src/app/components/FileUpload.vue

```diff
@@ -80,32 +96,47 @@
             hasError: function () {
                 return this.error.length > 0
             },
-            showRemove: function() {
+            showRemove: function () {
                 return this.hasError || this.valid
-            }
+            },
+            editsRequireConfirmation: mapGetterByName("stepper", "editsRequireConfirmation")
         },
         methods: {
-            handleFileSelect(_: Event, files: FileList | null) {
```

Comment:
I've removed this slightly weird thing we were doing of optionally passing a file list to this method. It was used only for testing, but since you innovated a much better way of testing file inputs it was actually obsolete. I updated all the existing tests to use your technique.

Reply:
Are we not going to have a future situation where people might want to upload multiple files of a type? Or are they always going to be zipped? I've kind of lost track of that whole debate...

Reply:
No we've gone with zip of zips!

Reply:
(But in any case more than this would have to change to support multiple)